docs/sphinx/releases/whats_new_1_9_0.rst

```diff
@@ -13,19 +13,176 @@
 
 General changes
 ===============
-
+- LCI parcelport was optimized.
+- HPX namespaces collapsed under `hpx::*`. `hpx::parallel` and `hpx::util` are now just `hpx`.
+- Codebase was modernized.
```

Comment:
@hkaiser  This is internal. Should I remove it?

Reply:
@gonidelis, could you please work with @dimitraka to create a documentation page that lists the namespace changes. We could then link to that page from the release notes.

Reply:
> This is internal. Should I remove it?

Nah, this is a QoI issue, let's list.

Reply:
> @gonidelis, could you please work with @dimitraka to create a documentation page that lists the namespace changes. We could then link to that page from the release notes.

Sure, where in the docs do you think this page fits best?

Reply:
Somewhere next to the release notes, perhaps? I'm not sure - what do you think?

Reply:
Under /releases we create a new_namespaces_1_9_0.rst file?

Reply:
please check https://github.com/STEllAR-GROUP/hpx/pull/6143